JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -108,7 +108,9 @@ private async Task<bool> CallMessageHandler(Message message)
 
             if (handler == null)
             {
-                return true;
+                _log.LogError($"Handler for message of type '{message.GetType().FullName}' not found in handler map. Returning message to queue.");
```

Comment:
I'd suggest using structured logging, rather than string interpolation.

```suggestion
                _log.LogError("Handler for message of type '{MessageTypeName}' not found in handler map. Returning message to queue.", message.GetType().FullName);
```